packages/trader/src/AppV2/Components/TradeParameters/Duration/container.tsx

```diff
@@ -17,6 +17,7 @@ const DurationActionSheetContainer = observer(
         const [toggle_date_picker, setToggleDatePicker] = useState<boolean>(false);
         const [current_gmt_time, setCurrentGmtTime] = useState<string>('');
         const [is_wheelpicker_loading, setIsWheelPickerLoading] = useState<boolean>(false);
+        const show_duration_chips = !(duration_units_list.length === 1 && duration_units_list[0].value === 't');
```

Comment:
i dont think second part of the check is necessary. what if we have only one timeframe thats not tick.

Reply:
For few trade types we get duration_units_list with length 1 but we need to add endtime there . 
for the cases we don't need endtime,I think we will upgrade this condition

Reply:
That for the case, when only 'days' is available and we need to add also chip for endtime. I just want to save Ahmad logic